docker/worker/start.sh

```diff
@@ -6,6 +6,18 @@ then
     echo "${TURBINIA_CONF}" | base64 -d > /etc/turbinia/turbinia.conf
 fi
 
+if [ ! -z ${TURBINIA_OUTPUT_DIR} ] && [ ! -d ${TURBINIA_OUTPUT_DIR} ]
+then
+  sudo mkdir -p ${TURBINIA_OUTPUT_DIR}
+  sudo chown -R turbinia:turbinia ${TURBINIA_OUTPUT_DIR}
```

Comment:
-R probably redundant here..(and in other places where we pass in a full folder and check if it exists)

Reply:
Removed the -R flags.